x/auth/client/cli/tx_sign.go

```diff
@@ -230,58 +231,49 @@ func makeSignCmd() func(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		// if --signature-only is on, then override --append
```

Comment:
we should keep this comment, it's the center of the debate https://github.com/cosmos/cosmos-sdk/pull/8106/files#r538193164

Reply:
1. This comment was in a wrong place (or the wording should be changed)
2. The comment is not relevant because signature-only and overwrite flags are independent now.

Reply:
Yeah I agree. It's outdated.